pkg/mimir/mimir_test.go

```diff
@@ -464,6 +464,18 @@ func TestConfigValidation(t *testing.T) {
 			},
 			expectAnyError: true,
 		},
+		{
+			name: "should pass if ingester ring has token generation strategy set to spread-minimizing, and target is not ingester",
+			getTestConfig: func() *Config {
+				cfg := newDefaultConfig()
+				_ = cfg.Target.Set("distributor")
```

Comment:
Should we add test for ingester that will throw error for invalid zones naming when spead-minimizing is enabled?